src/Php/PhpVersionProvider.php

```diff
@@ -51,9 +52,20 @@ public function provide(): string
         return PHP_VERSION;
     }
 
-    public function isAtLeastPhpVersion(string $phpVersion): bool
+    /**
+     * @param string|int $phpVersion
+     */
+    public function isAtLeastPhpVersion($phpVersion): bool
```

Comment:
make allow both string and int to avoid bc break in custom rector class that uses it.

Reply:
I think we can go for `int` here, as next version will be 0.9 and BC break is expected.

Reply:
implemented